src/celengine/processoctree.cpp

```diff
@@ -0,0 +1,251 @@
+
+#include "star.h"
+#include "deepskyobj.h"
+#include "astrooctree.h"
+#include "processoctree.h"
+
+using namespace Eigen;
+
+static constexpr double SQRT3 = 1.732050807568877;
```

Comment:
static is unneeded in such expressions. `constexpr` means "this can be evaluated at compile time", in case of numbers it's like C macros but better.